### PR TITLE
milestones, plans, runs, shared_steps api

### PR DESCRIPTION
**Pull request by abelharisov**

### CODE REVIEW

File: schemas/Attachment.hash.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  url:
+    type: string
+    format: uri
+
+
+              'hash' => $attachment->hash,
+              'size' => $attachment->size,
+              'mime' => $attachment->mime,
+              'filename' => $attachment->filename,
+              'url' => get_attachment_url($attachment, $team->hash)
```
Comment (kamilsk) [inline schemas/Attachment.hash.yaml:18]: wat?

File: schemas/Milestone.create.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    type: string
+  status:
+    type: string
+    enum:
+      - completed
+      - active
+  due_date:
+    type: integer
+    description: unix timestamp
+
```
Comment (kamilsk) [inline schemas/Milestone.create.yaml:15]: extra new line

**Review by kamilsk — CHANGES_REQUESTED** [review]

File: schemas/Attachment.hash.yaml (hunk 1 above)
Comment (abelharisov) [reply schemas/Attachment.hash.yaml]: fixed

File: schemas/Milestone.create.yaml (hunk 2 above)
Comment (abelharisov) [reply schemas/Milestone.create.yaml]: fixed

**Pull request merged**